lib/cputemp/advertisement.py

```diff
@@ -134,12 +136,19 @@ def register_ad_error_callback(self, err):
     def register(self):
         bus = BleTools.get_bus()
         adapter = BleTools.find_adapter(bus)
+        if not adapter:
+            logger.error("No Bluetooth Adapter")
```

Comment:
These error messages could be more clear for, especially for support agents who are not familiar with the code. The consequence of not having bluetooth adapter is not clear. Basically, I think the error should say `Advertisement not started: No Bluetooth adapter.`.